.github/workflows/pr-go.yaml

```diff
@@ -0,0 +1,139 @@
+name: pr-go
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "ui/web-v2/**"
+      - "python/**"
+      - "**/**.md"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  PROTOC_VERSION: 3.18.1
+  BUILDIFIER_VERSION: 5.1.0
+  GO_VERSION: 1.17.13
+  GOLANGCI_LINT: v1.46.2
```

Comment:
We use [golangci_lint@v1.39.0](https://github.com/bucketeer-io/bucketeer/blob/ef7198c15522e3f3e02acd93c425239d78185098/Makefile#L51) now.
I think we can update the one in Makefile.

Reply:
I have updated it to 1.47.2.
Thanks!
https://github.com/bucketeer-io/bucketeer/actions/runs/3231610977